signinapp/templates/event_stats.html.jinja2

```diff
@@ -2,6 +2,12 @@
 {% from 'bootstrap5/utils.html' import render_static %}
 {% block title %}Event {{event.name}}{% endblock %}
 {% block content %}
+<script>
+  function copyRegistrationLink() {
+    navigator.clipboard.writeText("{{registration_url}}");
+    alert("Copied: {{registration_url}}");
```

Comment:
Let's not use `alert`... I'd be fine with using another toast method like we do on the scan page though.